.dev/assets/shared/css/amp/amp.css

```diff
@@ -0,0 +1,24 @@
+body[class*="amp"] .primary-menu .menu-item-has-children:hover button.dropdown-toggle.toggled-on + ul,
```

Comment:
Question: why `body[class*="amp"]` and not just `body.amp`?

Reply:
@westonruter I didn't see a straight `amp` class on the body.
`amp-mode-touch home page-template-default page page-id-701 custom-background wp-custom-logo wp-embed-responsive theme-go woocommerce-no-js is-style-playful has-header-1 has-footer-3 has-comments-open has-header-background has-footer-background has-page-titles singular menu-is-open`

Reply:
Oh, I see.

In that case I actually recommend adding that as a body class manually.

```php
add_filter( 'body_class', function( $classes ) {
    if ( is_amp() ) {
        $classes[] = 'amp';
    }
    return $classes;
} );
```

That should end up being more reliable and it won't depend on the AMP framework to boot up to add the classes in order to apply the style rules.